components/brave_wallet/browser/eth_tx_controller.cc

```diff
@@ -153,7 +153,83 @@ void EthTxController::AddUnapprovedTransaction(
     return;
   }
 
-  EthTxStateManager::TxMeta meta(std::make_unique<EthTransaction>(*tx));
+  auto tx_ptr = std::make_unique<EthTransaction>(*tx);
+
+  // Use default gas price and limit for ETHSend.
+  if (tx_ptr->data().empty()) {  // ETHSend
+    if (!tx_ptr->gas_limit())
+      tx_ptr->set_gas_limit(21000);
```

Comment:
Pls add a constant for this value in `components/brave_wallet/browser/brave_wallet_constants.h`
`kDefaultSendEthGasLimit` or something like that

Reply:
Fixed in https://github.com/brave/brave-core/pull/10131/commits/642734ae3d2a7c4e088c8274d06d34db7bbd2986